boards/arm/frdm_kl46z/Kconfig.board

```diff
@@ -0,0 +1,11 @@
+# Kconfig - FRDM-KL25Z board
+#
+# Copyright (c) 2017, NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+#
```

Comment:
Please shorten to this:

```
# FRDM-KL25Z board

# Copyright (c) 2017, NXP
# SPDX-License-Identifier: Apache-2.0
```

See https://github.com/zephyrproject-rtos/zephyr/pull/20280.